meetings/2017-03-29.md

```diff
@@ -0,0 +1,188 @@
+* Ali Ijaz Sheikh @ofrobots (CTC)
+* Seth Thompson @s3ththompson (observer/Google)
+* MichaëZasso @targos (CTC)
+* Trevor Norris @trevnorris (CTC)
+* Rich Trott @Trott (CTC)
+
+
+## Standup
+
+* Anna Henningsen @addaleax (CTC)
+  * Issues & PR review
+  * ABI-stable native module API review in particular
+* Bradley Meck @bmeck (observer/GoDaddy/TC39)
+  * CC port for ESM underway
+  * onboarded Guy Bedford from SystemJS to help
+* Сковорода Никита Андреевич @ChALkeR (CTC)
+  * Mostly issue and PR cNothing worth mentioning
+* Colin Ihrig @cjihrig (CTC)
+  * Issues and PRs.
+* Evan Lucas @evanlucas (CTC)
+  * nothing of note
+* Franziska Hinkelmann @fhinkel (CTC)
+  * PR backlog
+* Jeremiah Senkpiel @Fishrock123 (CTC)
+  * Promise Unhandled rejection GC new pr thing https://github.com/nodejs/node/pull/12010
+* James M Snell @jasnell (CTC)
+  * landing PRs getting ready for 8.0.0
+* Michael Dawson @mhdawson (CTC)
+  * Away last week at Devoxx conference
+  * Build WG meeting
+  * AIX/disk speed discussions
+  * N-API review/work
+  * foundation npm user
+  * misc review/comment/land
+* Brian White @mscdex (CTC)
+  * _Finally_ finished next set of `fs` optimizations
+  * Updated/Rebased own open PRs
+  * Commented on issues, reviewed PRs
+* Ali Ijaz Sheikh @ofrobots (CTC)
+  * not too much of note
+* Seth Thompson @s3ththompson (observer/Google)
+  * v8 team working on landing TurboFan + Ignition in V8 5.9 
+* MichaëZasso @targos (CTC)
+  * Nothing of note
+* Trevor Norris @trevnorris (CTC)
+  * asynchooks PR
+* Rich Trott @Trott (CTC)
+  * lots of little things, but let.s talk about Buffer right now...
+
+## Agenda
+
+### nodejs/node
+
+* buffer: auto random fill Buffer(num) and new Buffer(num) [#11808](https://github.com/nodejs/node/pull/11808)
+* buffer: discuss future direction of Buffer constructor API [#9531](https://github.com/nodejs/node/issues/9531)
+
+## Previous Meeting Review
+
+* buffer: discuss future direction of Buffer constructor API [node#9531](https://github.com/nodejs/node/issues/9531)
+* module: add support for abi stable module API [node#11975](https://github.com/nodejs/node/issues/11975)
+
+## Minutes
+
+### buffer: auto random fill Buffer(num) and new Buffer(num) [#11808](https://github.com/nodejs/node/pull/11808) and buffer: discuss future direction of Buffer constructor API [#9531](https://github.com/nodejs/node/issues/9531)
+
+* Rich started by framing the current state of the issue. Some
+  consensus that we should do something.  Had looked like 
+  option to do opt-in deprecate flag was non-controversial, but
+  seems like there is some questioning/objections.  
+* We may need to start with if we are going to state that it 
+  will be deprecated at some point as that affects people.s
+  decisions on the other issues.
+* James: we need 3 distinct votes
+  * commit to runtime deprecation and timeframe
+  * do we put in pending deprecation
+  * do we random fill or zero fill by default, and do we backport
```

Comment:
That's an interesting point I hadn't thought about before. Would you mind mentioning it in https://github.com/nodejs/node/pull/11808?